pkg/configtx/application.go

```diff
@@ -111,12 +111,11 @@ func (c *ConfigTx) AddAnchorPeer(orgName string, newAnchorPeer Address) error {
 }
 
 // RemoveAnchorPeer removes an anchor peer from an existing channel config transaction.
-// The removed anchor peer and org it belongs to must both already exist.
+// Specifying an anchor peer or organization name that does not exist in the application
+// ConfigGroup of the channel config will not return a error.
+// Removal will panic if application group does not exist.
```

Comment:
Will also panic if application org group does not exist when attempting to access Values on line 122